Bio/motifs/matrix.py

```diff
@@ -10,58 +10,25 @@
 
 import math
 import platform
+import numpy as np
```

Comment:
Continuing my query about the numpy import, style aside (#2391) can you use this please:

```python
try:
    import numpy as np
except ImportError:
    from Bio import MissingPythonDependencyError
    raise MissingPythonDependencyError(
        "Install NumPy if you want to use Bio.motifs.matrix.")
```

This should fix the failures on Travis on the target which deliberately does not have numpy installed. See #2390 for the background to this.

Reply:
Done.

Regarding style, I wasn't sure whether there was any specific style choice in Biopython and just went with the universally used convention...